service/src/main/java/bio/terra/cbas/dao/MethodDao.java

```diff
@@ -35,23 +35,28 @@ public MethodDao(NamedParameterJdbcTemplate jdbcTemplate) {
 
   public Method getMethod(UUID methodId) {
     String sql =
-        String.format(
-            "SELECT * FROM method %s WHERE method.method_id = :methodId",
-            METHOD_JOIN_GITHUB_METHOD_DETAILS);
-    return jdbcTemplate
-        .query(sql, new MapSqlParameterSource("methodId", methodId), new MethodMapper())
-        .get(0);
+        "SELECT * FROM method %s WHERE method.%s = :methodId AND %s = false"
```

Comment:
my vote would be to leave those adjustments for a followup ticket, after we map out all of the implications, because denying knowledge of it altogether achieves our immediate needs